emails/utils.py

```diff
@@ -19,14 +20,15 @@
 import jwcrypto.jwk
 import markus
 import logging
+import urllib.parse
+from urllib.parse import urlparse
```

Comment:
nit: import `quote_plus` directly

```suggestion
from urllib.parse import quote_plus, urlparse
```

This will change the usage below as well